app/views/admin/school_groups/_form.html.erb

```diff
@@ -24,8 +24,8 @@
         The following defaults are used to pre-populate values when creating a school. Changing them here will not affect areas for schools currently in this group.
       </p>
       <div class="form-group">
-        <%= f.label :default_calendar_area_id, 'Default Calendar Area' %>
-        <%= f.select :default_calendar_area_id, options_from_collection_for_select(CalendarArea.order(:title), 'id', 'title', @school_group.default_calendar_area_id), {include_blank: true}, { class: 'form-control' } %>
+        <%= f.label :default_template_calendar_id, 'Default Template Calendar' %>
+        <%= f.select :default_template_calendar_id, options_from_collection_for_select(Calendar.template.order(:title), 'id', 'title', @school_group.default_template_calendar_id), {include_blank: true}, { class: 'form-control' } %>
```

Comment:
Not sure about the `template` scope on the model. I think I'd prefer it to be clear that we're using regional calendars here.

Reply:
I agree - that shouldn't be template, should be regional